custom/intrahealth/tests/utils.py

```diff
@@ -36,6 +36,9 @@ def setUp(self):
         user.is_active = True
         self.user = user
 
+    def tearDown(self):
```

Comment:
Would it be possible to change this to a `setUpClass`/`tearDownClass` , so that less setup and tear down need to happen? Fine to do in another PR, since this already exists.

Also when that is changed, you should clean up the domain as well